test/acceptance/framework/config/config_test.go

```diff
@@ -76,6 +76,15 @@ func TestConfig_HelmValuesFromConfig(t *testing.T) {
 				"global.openshift.enabled": "true",
 			},
 		},
+		{
+			"sets enablePodSecurity policies when -enable-pod-security-policies is set",
```

Comment:
not a real comment but it was funny to read `sets enable pod security policies when enable pod security policies is set`. kinda reads like a tongue twister

Reply:
yeah, it does sound weird! rephrased!